scripts/core/superdesk-api.d.ts

```diff
@@ -559,6 +559,17 @@ declare module 'superdesk-api' {
         preview?: boolean;
     }
 
+    export interface IFeedingServiceField extends IBaseRestApiResponse {
+        id?: string;
```

Comment:
It shouldn't extend `IBaseRestApiResponse` - this type is only API response for a single resource like `users/x`. These responses have `_etag`, `_created`, `_updated`, `_id` fields.

Can you also put this interface in the file where you are using it instead of here? Here we put only interfaces that need to be exposed to extensions.